content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -1777,23 +1784,9 @@ const result = await prisma.user.findMany({
 
 #### Generated types for `select`
 
-The following examples demonstrate how to use the [`validator`](/orm/prisma-client/type-safety/prisma-validator) with `select`:
-
-- `SelectAndInclude`
-- `UserSelect`
+The following example demonstrates how to use the [`validator`](/orm/prisma-client/type-safety/prisma-validator) with `select`:
```

Comment:
Intentional changes here?

Reply:
Yes, `SelectAndInclude` example does not really make sense - it *requires* `select` and `include` to be used together which in turn is not allowed by either type- or runtime- validation in the client. It's an internal type which used for validation (basically, only to say "If user's input looks like that then complain")